arguments/partialuser.js

```diff
@@ -0,0 +1,31 @@
+const { Argument } = require('klasa');
+
+module.exports = class extends Argument {
+
+	async run(arg) {
+		const user = await this.resolveUser(arg, this.client.users);
+		if (user) return user;
+		throw `Please specify a valid user.`;
+	}
+
+	resolveUser(text, users, caseSensitive = false, wholeWord = false) {
```

Comment:
caseSensitive and wholeWord aren't used here, remove them and all code relating to this

Reply:
done